### DISCUSSION TITLE
Annoying context menu documentation information flickers!

### DISCUSSION THREAD

**Discussion by tiberiumboy** [zed-industries/zed#8744] [Feature Requests]
Trying out Zed for the first time on MBA M2,
Opened my rust project, and reading through the object description, the context menu would be flashing periodically. Other elements do not exhibit this behavior, as can seen in the video below
Needless enough to express my frustration through the cursor in this short video, please let me know what else I need to provide. 
Using Zed 0.124.7
MacBook Air M2 2022

https://github.com/zed-industries/zed/assets/8661186/31eb85a1-e853-4eac-95e8-356eac3582d5

**Comment by Moshyfawn** [comment]
This is a known issue tracked in #8579.

**Reply by tiberiumboy** [reply]
Ouch! Glad I'm not alone on this one!

**Reply by Moshyfawn** [reply]
👌
